tutorials/iot/using-intel-realsense-sdk.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: using-intel-realsense-sdk
 summary: We are going to illustrate how to build some Intel RealSense SDK samples on an ubuntu desktop. You will be able to hook up on your Intel Joule and build the samples, explore some of the code and try them!
-categories: snap
+categories: iot
```

Comment:
Should this also be `packaging`? As `iot` has been removed from the filters?

Reply:
@anthonydillon iot is introduced in this PR, so this change (and the others you pointed out) is accurate, packaging is new as well, but is used for non-iot focused snapping.

Reply:
Sorry, your right. I miss read the additions.